package.json

```diff
@@ -75,6 +75,7 @@
     "eslint-plugin-jest": "26.5.3",
     "eslint-plugin-react": "7.30.1",
     "eslint-plugin-react-native": "4.0.0",
+    "eslint-plugin-react-native-a11y": "3.2.1",
```

Comment:
this officially only supports eslint 7 but we are using 8, therefore `make install` fails. I guess the two options we have are 
1. downgrade eslint
2.  force it to use eslint 8 and live with the risk
You can add this to package.json to make it work
```
 "overrides": {
    "eslint-plugin-react-native-a11y": {
      "eslint": "$eslint"
    }
  },
```